### PR TITLE
task3

### PR DESCRIPTION
**Pull request by yana-may25**

### CODE REVIEW

File: src/task3/task3.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function isEven(n)
+    {
```
Comment (MrOlegKravchenko) [inline src/task3/task3.js:2]: Function writing pattern please

Comment (MrOlegKravchenko) [reply src/task3/task3.js:2]: This function should be much shorter.
Recursion is a key and you use it well !
Best to write in 7 ish lines

Comment (yana-may25) [reply src/task3/task3.js]: Oh, ur patience's gonna be rewarded

File: src/task3/task3.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function isEven(n) {
```
Comment (MrOlegKravchenko) [inline src/task3/task3.js:1]: Exactly!
But there should be same pattern, everywhere:
```
function isEven(n) {
    if (n === 1) {
        return false;
    } if (n === 0) {
        return true;
    } else {
        return n > 0 ? isEven(n - 2) : isEven(n + 2);
    }
};
```

Comment (yana-may25) [reply src/task3/task3.js:1]: ok, lesson learned, I suppose
but "Function writing pattern" was not immediately apparent to me, I'm afraid :(

Comment (MrOlegKravchenko) [reply src/task3/task3.js:1]: I am sorry I misguide you
Should be like this:
```
function isEven(n) {
    if (n === 1) {
        return false;
    } else if (n === 0) {
        return true;
    } else {
        return n > 0 ? isEven(n - 2) : isEven(n + 2);
    }
};
```

**Pull request closed without merging**